app/boards/shields/breeze/Kconfig.defconfig

```diff
@@ -0,0 +1,17 @@
+
+if SHIELD_BREEZE_LEFT
+
+config ZMK_KEYBOARD_NAME
+    default "Breeze"
+
+config ZMK_SPLIT_BLE_ROLE_CENTRAL
```

Comment:
```suggestion
config ZMK_SPLIT_ROLE_CENTRAL
```

Reply:
Fixed in https://github.com/zmkfirmware/zmk/pull/1312/commits/c937f3057f3d33d42ffd4adcbc0aaf65bdbe6dd3